Google.Api.Generator.Rest/CodeGenerator.cs

```diff
@@ -143,7 +143,10 @@ void InsertLine(string contentToFind, int offset, string extraLine)
         private static ResultFile GenerateProjectFile(PackageModel package)
         {
             var doc = package.GenerateProjectFile();
-            return new ResultFile($"{package.PackageName}/{package.PackageName}.csproj", doc.ToString());
+            // Take care of fine-grained line-break placement, and add a final line break for compatibility
+            // with existing generator (which really doesn't need to be changed for this).
+            string text = doc.ToString().Replace("  <!--linebreak-->", "") + Environment.NewLine;
```

Comment:
You are replacing `linebreak ` by and empty string? Should it be replaced by a new line instead?

Reply:
No, because the XComment is already formatted onto a line on its own. Sounds hokey, but it works. I'll clarify the comment though :)

Reply:
Ah yes, makes sense.